crypto/crypto_test.go

```diff
@@ -28,69 +28,94 @@ import (
 )
 
 func TestGeneratePrivateKey(t *testing.T) {
+	ecdsaAlgos := []crypto.SignatureAlgorithm{
+		crypto.ECDSA_P256,
+		crypto.ECDSA_secp256k1,
+	}
+
+	nonECDSAAlgos := []crypto.SignatureAlgorithm{
+		crypto.BLS_BLS12381,
+	}
+
```

Comment:
This is minor, but same as above, the separation is more about SDK-supported/non-SDK-supported rather than ECDSA/non-ECDSA.